main.css

```
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

We can also add action-kit styles to this file.

```suggestion
@tailwind utilities;

@import '@mobileaction/action-kit/dist/style.css';
```